release-engineering/role-handbooks/release-image-promotion.md

```diff
@@ -43,4 +44,5 @@ Examples:
 
 ```shell
 skopeo inspect docker://us.gcr.io/k8s-artifacts-prod/kube-apiserver:v1.19.0-rc.4 --raw
+skopeo inspect docker://k8s.gcr.io/kube-apiserver:v1.19.0-rc.4 --raw
 ```
```

Comment:
As discussed on Slack, this is mostly like no longer needed since krel does it for us. Should we remove this step?

Reply:
I'm in favor to remove it, because we have those checks already in place in krel. There may be other opinions.

Reply:
I've removed it. If there are any objections, I'll add it again.